state-chain/runtime/src/lib.rs

```diff
@@ -901,27 +901,12 @@ type PalletMigrations = (
 	pallet_cf_environment::migrations::VersionUpdate<Runtime>,
 	pallet_cf_environment::migrations::PalletMigration<Runtime>,
 	pallet_cf_funding::migrations::PalletMigration<Runtime>,
-	// pallet_cf_validator::migrations::PalletMigration<Runtime>,
-	pallet_cf_governance::migrations::PalletMigration<Runtime>,
-	pallet_cf_tokenholder_governance::migrations::PalletMigration<Runtime>,
 	pallet_cf_chain_tracking::migrations::PalletMigration<Runtime, Instance1>,
 	pallet_cf_chain_tracking::migrations::PalletMigration<Runtime, Instance2>,
 	pallet_cf_chain_tracking::migrations::PalletMigration<Runtime, Instance3>,
-	pallet_cf_vaults::migrations::PalletMigration<Runtime, Instance1>,
-	pallet_cf_vaults::migrations::PalletMigration<Runtime, Instance2>,
-	pallet_cf_vaults::migrations::PalletMigration<Runtime, Instance3>,
-	// pallet_cf_threshold_signature::migrations::PalletMigration<Runtime, Instance1>,
-	// pallet_cf_threshold_signature::migrations::PalletMigration<Runtime, Instance2>,
-	// pallet_cf_threshold_signature::migrations::PalletMigration<Runtime, Instance3>,
 	pallet_cf_broadcast::migrations::PalletMigration<Runtime, Instance1>,
 	pallet_cf_broadcast::migrations::PalletMigration<Runtime, Instance2>,
 	pallet_cf_broadcast::migrations::PalletMigration<Runtime, Instance3>,
-	pallet_cf_swapping::migrations::PalletMigration<Runtime>,
-	pallet_cf_lp::migrations::PalletMigration<Runtime>,
-	pallet_cf_ingress_egress::migrations::PalletMigration<Runtime, Instance1>,
-	pallet_cf_ingress_egress::migrations::PalletMigration<Runtime, Instance2>,
-	pallet_cf_ingress_egress::migrations::PalletMigration<Runtime, Instance3>,
-	pallet_cf_pools::migrations::PalletMigration<Runtime>,
```

Comment:
I think we should keep these here - we can unlink the individual upgrades but the 'framework' boilerplate can stay IMO.

Reply:
Well, but then we still compile it into the runtime and size stays the same. Then actually can just keep them. They are getting skipped anyway.

Reply:
Maybe you misunderstood the suggestion - see the other longer comment.

I think we can make things as developer-friendly as possible by just removing the v1 v2 etc migrations and keeping the boilerplate 'shell'.

This way adding a new migration again is simply adding a new vX migration, and you don't need to remember to add something to the runtime, uncomment the migration module declaration etc. 

Nothing extra will be compiled into the runtime - if there is no migration the implementation is just `()` or similar.